beacon-chain/db/kv/kv.go

```diff
@@ -25,12 +26,30 @@ const (
 	VotesCacheSize = 1 << 23
 	// NumOfVotes specifies the vote cache size.
 	NumOfVotes = 1 << 20
+	// ValidatorEntryCacheSize is set to 1MB since we expect ~100K validator entries (10 times the expected size)
```

Comment:
Why 100k?

Reply:
i see on an average ~100k registry validator entries for very state in mainnet. 
https://gist.github.com/jmozah/ab1abbe9d9035002e6cc1e4daa00f1fb